app/services/collators/disposable_income_collator.rb

```diff
@@ -57,17 +61,25 @@ def default_attrs
       }
     end
 
+    # TODO: this could just be delegated
     def fixed_employment_allowance
       @fixed_employment_allowance ||= disposable_income_summary.fixed_employment_allowance
     end
 
+    # TODO: this could just be delegated
     def employment_income_deductions
       @employment_income_deductions ||= disposable_income_summary.employment_income_deductions
     end
 
     def total_outgoings_and_allowances
-      net_housing_costs + dependant_allowance + child_care_bank + maintenance_out_bank + legal_aid_bank\
-      + @monthly_cash_transactions_total - fixed_employment_allowance - employment_income_deductions
+      net_housing_costs +
+        dependant_allowance +
+        child_care_bank +
+        maintenance_out_bank +
+        legal_aid_bank +
+        @monthly_cash_transactions_total -
+        fixed_employment_allowance -
+        employment_income_deductions
     end
```

Comment:
I think we need to make it a bit clearer what is going on  here - it seems to be taking only the bank transactions of child_care, maintenance_out and legal _aid, but using the summed total in @monthly cash_transactions as well.

Perhaps we'd be better off writing a method `monthly_bank_transactions` that adds those three together, and then just adding those in this method